lua/apisix/utils/batch-processor.lua

```diff
@@ -57,12 +57,12 @@ function execute_func(premature, batch_processor, batch)
         return
     end
 
-    local ok, err = batch_processor.func(batch.entries)
+    local ok, err = batch_processor.func(batch.entries, batch_processor.batch_max_size)
     if not ok then
+        core.log.error(fmt("Batch Processor[%s] failed to process entries: ",
```

Comment:
optimize: we should avoid to generate any new object , it will increase the load on the Lua GC and make APISIX performance worse.

good style: `core.log.error("Batch Processor[", batch_processor.name, "] failed to process entries: ", err)`